yt_dlp/extractor/teamcoco.py

```diff
@@ -76,126 +124,157 @@ class TeamcocoIE(TurnerBaseIE):
         }, {
             'url': 'http://teamcoco.com/israel/conan-hits-the-streets-beaches-of-tel-aviv',
             'only_matching': True,
-        }, {
-            'url': 'https://conan25.teamcoco.com/video/ice-cube-kevin-hart-conan-share-lyft',
-            'only_matching': True,
-        }
+        },
     ]
-    _RECORD_TEMPL = '''id
-        title
-        teaser
-        publishOn
-        thumb {
-          preview
-        }
-        tags {
-          name
-        }
-        duration
-        turnerMediaId
-        turnerMediaAuthToken'''
-
-    def _graphql_call(self, query_template, object_type, object_id):
-        find_object = 'find' + object_type
-        return self._download_json(
-            'https://teamcoco.com/graphql', object_id, data=json.dumps({
-                'query': query_template % (find_object, object_id)
-            }).encode(), headers={
-                'Content-Type': 'application/json',
-            })['data'][find_object]
 
     def _real_extract(self, url):
-        display_id = self._match_id(url)
+        display_id = self._match_id(url).replace('/', '_')
+        webpage = self._download_webpage(url, display_id)
+        data = self._search_nextjs_data(webpage, display_id)['props']['pageProps']['pageData']
+        info = merge_dicts(*[traverse_obj(
+            data, ('blocks', lambda _, v: v['name'] == name, 'props'),
+            get_all=False, expected_type=dict, default={}) for name in ('meta-tags', 'video-player', 'video-info')])
 
-        response = self._graphql_call('''{
-  %%s(slug: "%%s") {
-    ... on RecordSlug {
-      record {
-        %s
-      }
-    }
-    ... on PageSlug {
-      child {
-        id
-      }
-    }
-    ... on NotFoundSlug {
-      status
-    }
-  }
-}''' % self._RECORD_TEMPL, 'Slug', display_id)
-        if response.get('status'):
-            raise ExtractorError('This video is no longer available.', expected=True)
-
-        child = response.get('child')
-        if child:
-            record = self._graphql_call('''{
-  %%s(id: "%%s") {
-    ... on Video {
-      %s
-    }
-  }
-}''' % self._RECORD_TEMPL, 'Record', child['id'])
-        else:
-            record = response['record']
-        video_id = record['id']
+        thumbnail = traverse_obj(
+            info, (('image', 'poster'), {lambda x: urljoin('https://teamcoco.com/', x)}), get_all=False)
+        video_id = thumbnail.rpartition('=')[2] if thumbnail else display_id
```

Comment:
that's a weird way to get video_id

Reply:
not ideal, but with the site change now it's the only way to get the old video_id that users would have in their archives